### PR TITLE
Adding hindi movies to the list

### PR DESCRIPTION
**Pull request by an-mittal**
speak to harman If you have any questions.

And thanks!

### CODE REVIEW

File: scripts/index.js [hunk 1]
```diff
@@ -80,6 +80,9 @@ function fire() {
 		case 'movies':
 			sessionData = movieData.slice(0);
 			break;
+        case 'hindimovies':
```
Comment (harman28) [inline scripts/index.js:83]: Indentation @an-mittal, what

Comment (an-mittal) [reply scripts/index.js:83]: its correct. Github is showing it wrongly

Comment (harman28) [reply scripts/index.js:83]: @captn3m0 I'll add .editorconfig?

Comment (captn3m0) [reply scripts/index.js:83]: Yes please.

**Comment by captn3m0** [comment]
Thanks a lot @an-mittal :100: :1st_place_medal:

**Pull request merged**